.github/workflows/documentation.yml

```diff
@@ -8,7 +8,7 @@ permissions: read-all
 
 jobs:
   Build-Documentation:
-    runs-on: [self-hosted, A100]
+    runs-on: [a100-runner-set]
```

Comment:
Oh I see, it used to run on job `A100`.  So no change if you run it on `a100-runner-set`!

Reply:
It's indeed slow...